code/modules/admin/verbs/forcecryo.dm

```diff
@@ -0,0 +1,10 @@
+/proc/forcecryo(mob/target)
+    var/turf/T = get_turf(target)
+    new /obj/effect/temp_visual/tornado(T)
+    sleep(20)
+    var/obj/machinery/cryopod/C = new /obj/machinery/cryopod(T)
```

Comment:
any real reason to initialize it on a turf instead of just keeping it as a temporarly reference to use its procs? Pretty sure that should just work fine instead of actually spawning it on a turf might cause some mess to where the ghost spawns but thats the worst that happens and won't really matter if there is none. If you do that you also won't need the invisibility var sets at all not that i understand why you set the device and the mob invisible

Reply:
and if the ghost spawns is the problem just overwrite the object so its a child and fix the proc for the despawn to spawn the ghost back at its original location or something alike.